Tests/PHPCI/Helper/AnsiConverterTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace PHPCI\Helper\Tests;
+
+use PHPCI\Helper\AnsiConverter;
+use Prophecy\PhpUnit\ProphecyTestCase;
+
+class AnsiConverterTest extends ProphecyTestCase
```

Comment:
If you extend PhtopecyTestCase for sure (but in this case you are not mocking anything), or if you use only PHPUnit only if the version used is >=4.5 (see [announcement](https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-4.5.0#new-features) )